ooniprobe/View/TestResults/TestResultsTableViewController.m

```diff
@@ -35,7 +36,12 @@ -(void)testFilter:(SRKQuery*)newQuery{
         if ([dic objectForKey:key])
             arr = [[dic objectForKey:key] mutableCopy];
         [arr addObject:current];
-        [dic setObject:arr forKey:key];
+        if (key == nil){
+            //log object
+            [CrashlyticsKit recordError:[NSError errorWithDomain:@"key_nill" code:0 userInfo:[current dictionary]]];
```

Comment:
`key_nil` (with only one `l`)?

Reply:
I would assume `userInfo` will be serialised in a way in which we don't see issues?

Reply:
- `nil` is an alternative way to say `NULL` in obj-c
- Good question, let me set date to nil and send that object